scripts/util/metadata.py

```diff
@@ -6,11 +6,16 @@
 seq_file_1 = "seq_file_1"
 seq_file_2 = "seq_file_2"
 
-EXPECTED_HEADERS = {SAMPLE_ID, seq_file_1, seq_file_2}
 ILLUMINA = "illumina"
 ONT = "ont"
 UNKNOWN = "unknown"
 
+EXPECTED_HEADERS = {
```

Comment:
Not sure what the standard is for the business logic, but everything is typed on the testing side, even globals. It avoids ambiguity and lets `mypy` pick up on misused objects.

Reply:
Yes, I don't fancy trying to add typing to the whole pipeline, and fix any other formatting and linting to make hooks pass right now.